libs/saksbehandling-common/src/main/kotlin/feilhaandtering/Utils.kt

```diff
@@ -11,6 +11,7 @@ fun <T : Any> checkNotNullOrThrowException(
     }
 }
 
+// Todo endre til if(value)
```

Comment:
TODO? Er det noe mer å gjøre her da? if (!value) ser jo riktig ut 😄

Reply:
Det er riktig men bruken burde vært uten negering, men droppa det pga risiko for feil